libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -170,6 +179,16 @@ AP_BattMonitor::read()
             drivers[i]->update_resistance_estimate();
         }
     }
+
+    if (get_type() != BattMonitor_TYPE_NONE) {
```

Comment:
change `BattMonitor_TYPE_NONE` to `AP_BattMonitor_Params::BattMonitor_TYPE_NONE` so it compiles :)

Reply:
This was opened before the params stuff went in, and I knew I had to rebase it, just forgot to touch that field :P